constants.py

```diff
@@ -3,13 +3,15 @@
 
 config = dotenv_values('.env')
 
-POSTGAME = "Yay!" # "Yay!" or "Boo!" or None
```

Comment:
It's a goofy way of doing this but keep the comment for now since the code looks for those exact strings.